src/scripts/downloadPages.ts

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env node
+
+import { downloadPages } from '../lib/downloadPages';
+import { readDir, readFile } from '../lib/helpers';
+
+import * as path from 'path';
+
+const distDirectory = path.resolve(process.cwd(), 'downloaded');
```

Comment:
Ah, I was wondering if `path.join` worked like this. Now I know it doesn't. Will use `path.resolve` next time.

Reply:
I think both work here, although they behave differently when the first segment starts with a slash (https://stackoverflow.com/questions/35048686/difference-between-path-resolve-and-path-join-invocation)

Reply:
Yeah. But I think it's generally better to use `resolve`. I used `join` in `validate-filenames` so now you can't specify an absolute path to the config file there.